src/Forms/DefaultFormFactory.php

```diff
@@ -47,6 +47,15 @@ public function getForm(RequestHandler $controller = null, $name = FormFactory::
         $validator = $this->getFormValidator($controller, $name, $context);
         $form = Form::create($controller, $name, $fields, $actions, $validator);
 
+        // Sudo mode
+        if (is_a($context['Record'], DataObject::class)) {
```

Comment:
Scenario is ... ?

Possibly should get rid of this as it may impact frontend forms